src/DependencyInjection/SymfonyRequestIdExtension.php

```diff
@@ -6,14 +6,14 @@
 
 use DR\SymfonyRequestId\EventSubscriber\CommandSubscriber;
 use DR\SymfonyRequestId\EventSubscriber\MessageBusSubscriber;
-use DR\SymfonyRequestId\EventSubscriber\RequestIdSubscriber;
+use DR\SymfonyRequestId\EventSubscriber\TraceIdSubscriber;
 use DR\SymfonyRequestId\Generator\RamseyUuid4Generator;
 use DR\SymfonyRequestId\Generator\SymfonyUuid4Generator;
-use DR\SymfonyRequestId\Monolog\RequestIdProcessor;
-use DR\SymfonyRequestId\RequestIdGeneratorInterface;
-use DR\SymfonyRequestId\RequestIdStorageInterface;
+use DR\SymfonyRequestId\Monolog\TraceIdProcessor;
+use DR\SymfonyRequestId\IdGeneratorInterface;
+use DR\SymfonyRequestId\IdStorageInterface;
 use DR\SymfonyRequestId\SimpleIdStorage;
-use DR\SymfonyRequestId\Twig\RequestIdExtension;
+use DR\SymfonyRequestId\Twig\TraceIdExtension;
 use RuntimeException;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Exception\LogicException;
```

Comment:
Rename class

Reply:
My idea was to do this in a separate PR, where the bundle is renamed and possibly also the git package